workflow/python/barebones/v1/README.md

```diff
@@ -0,0 +1,40 @@
+# workflow-python-barebones-v1
+This is the smallest Python image that allows users to run workflows.
+
+## Image Details
+### Base Image
+This image uses `python:3.8.13` as its base which is maintained by [the Docker Community](https://github.com/docker-library/python).
+
+### OS and other basic details
+```
+Debian         11 (bullseye)
+Linux Kernel   5.10.104-linuxkit
+Python         3.8.13
+```
+
+### Linux Packages Installed
+```
+aws-cli        2.2.5
+git            3.8.8
+```
+
+### Python Libraries Installed
+```
+boto3          1.22.7
+psycopg2       2.9.3
+sqlalchemy     1.4.36
+```
+
+### Build Arguments
+The Dockerfile expects the following build arguments:
+- `PEAK_USER_ID`: This is the default user that the workflow step runs with when this image is used in the Peak platform. On the Peak platform, this value must be `8877`.
+
+## Building the Image
```

Comment:
We should describe how we build the the image on the platform first. Building locally should be second (if at all).

Reply:
Hey, I was thinking of adding how to build an image on the platform at a commonplace - README for the repo or a separate doc altogether rather than copy/pasting the same detail in each image's README.

Have added this to each image as each can have a slightly different command required to build them - may be a separate build argument or something.